client/pages/StatUnits/View/tabs/Activity.jsx

```diff
@@ -1,18 +1,31 @@
 import React from 'react'
 import { shape, func, arrayOf, string } from 'prop-types'
 import { Header } from 'semantic-ui-react'
-
-import { ActivitiesField } from '/client/components/fields'
+import { ActivitiesList } from '/client/components/fields'
 import styles from './styles.scss'
 
-const Activity = ({ data, localize, activeTab }) => (
-  <div>
-    {activeTab !== 'activity' && (
-      <Header as="h5" className={styles.heigthHeader} content={localize('Activity')} />
-    )}
-    <ActivitiesField name="activities" value={data} localize={localize} readOnly />
-  </div>
-)
+class Activity extends React.Component {
```

Comment:
Here you are rewriting from Functional Component to Classical React.
Why?
The only practical change seem to be the rename of ActivitiesField to ActivitiesList.

Reply:
Will look into it shortly.

Reply:
@jhf Good point! I re-wrote it back to using hooks and it's working as it should.

Reply:
@jhf I also re-wrote FieldGroup and FormSection to use hooks again and everything is working fine now.